src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithAllFeatures.cs

```diff
@@ -61,6 +61,7 @@ public void It_publishes_the_project_correctly(string targetFramework, string []
             ""System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization"": false,
             ""System.Diagnostics.Debugger.IsSupported"": true,
             ""System.Diagnostics.Tracing.EventSource.IsSupported"": false,
+            ""System.Diagnostics.Metrics.Meter.IsSupported"": false,
```

Comment:
```suggestion
            ""System.Diagnostics.Metrics.Meter.IsSupported"": false,
            ""System.Diagnostics.Tracing.EventSource.IsSupported"": false,
```

(nit) the comment above says "keep this list sorted".